lib/known.rb

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env ruby
 
-require_relative('../conf.rb' )
-
-cmd_path = $conf[:dir][:root] + '/cmd'
+root_path = File.expand_path('..', File.dirname(__FILE__))
```

Comment:
This is being called in each lib often. Would there be any way to store this path somewhere globally and just reference it here?